packages/shared/src/hooks/useFeed.ts

```diff
@@ -138,6 +139,15 @@ export default function useFeed<T>(
     {
       refetchOnMount: false,
       ...options,
+      retry: (failureCount, error) => {
```

Comment:
Is there any way to add globally?

Reply:
Yeah also, but for now it's only feed and want to raise a specific `onError` here in the next iteration.
So not seeing a point to do it now if that makes sense?

Reply:
Noticed we can just overwrite it then and use it seperate :)

Reply:
but we also limit the number of retries. i'm afraid in case of an issue frontend will hammer api other endpoints

Reply:
Limit to? I mean we can remove retries all together?